_posts/2018-01-19-event-lineup.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "The 2018 Rust Event Lineup"
+author: Rust Community
+description: "Multitude of Rust event are happening this year; join us at one near you!"
+---
+
+Every year there are multiple Rust events around the world, bringing together the community.
+
+### February 1-3: Mozilla Rust Roadshow Brasil 2018
+
+The [Rust Roadshow][roadshow] is a series of workshops for Rust.
+Learn Rust with the community in several cities across Brazil.
+
+Several workshops are already scheduled for the beginning of February, with hopefully more to come.
+Find all information on [the Roadshow website][roadshow].
+
+[roadshow]: https://rust-br.github.io/2018-roadshow/
+
+### February 3-4: FOSDEM - Rust Dev Room
+
+After a very successful Bird of Feathers session for Rust last year at FOSDEM,
```

Comment:
is there a link to explain what "Bird of Feathers" is? might be useful here, i am not sure what that means and other might feel the same

Reply:
It's "Birds of a feather" https://en.wikipedia.org/wiki/Birds_of_a_feather_(computing)